site/pages/review-disruption/[disruptionId].page.tsx

```diff
@@ -135,6 +135,159 @@ const ReviewDisruption = ({
                   .consequenceIndex + 1
             : 0;
 
+    const getRows = (consequence: Consequence) => {
```

Comment:
Do we also need to make these changes on the disruption detail page as well?

Reply:
done